spec/support/vcr_cassettes/decision_review/HLR-CREATE-RESPONSE-503_V1.yml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
I just copied the 200 response and changed the status to 503 to simulate the error getting raised for the test I added 🤷‍♀️